### DISCUSSION TITLE
Is it possible to merge two storage paths to one, if so, how should one go about doing it?

### DISCUSSION THREAD

**Discussion by cwhiggins** [filecoin-project/lotus#5496] [Storage Provider] [answered]
I’ve got to merge 2 lotus storages path . Can I do the following without problem :

1/ Turn off the miner
2/ copy the sectors file to the new location
3/ Disable the old storage in storage.json
4/ Restart the miner  ?

A question posted by @s0nik42, I am also curious to the answer @Magik6k, copied from minerX, reposted in mainnet on Slack

**Comment by magik6k** [comment, marked as answer]
Yes, this should work, the sector index in rebuilt on lotus-miner/worker startup.

You can even avoid some downtime by copying sectors first, then stopping the miner, editing storage.json, and starting it back up

**Reply by pioter83** [reply]
Shouldn't we change all "true" to "false" in miner's config.toml in section [Dealmaking] - to stop dealmaking, before copying the sectors to new location?

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
you should avoid writing to both storage locations - unrelated to deals

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
https://docs.filecoin.io/mine/lotus/miner-lifecycle/#changing-storage-locations is the palce this is documented - just for having it mentioned here
